modules/ROOT/pages/6.6.1-release-notes.adoc

```diff
@@ -122,8 +122,15 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 === On Safari and Firefox browsers, scroll positions were not always maintained when updating the content of a `streamContent: true` iframe dialog component
 //#TINY-10078
 
-// CCFR here.
+On each content update for iframe dialog components with `streamContent: true` set, the iframe is expected to automatically scroll to the bottom if the pre-update scrolled position is already at the end of the scrollable area. It is also expected, however, that the end-user can freely scroll through the iframe as content updates occur.
+
+When Chrome is the host browser, scroll positions are natively maintained after `document.write()`, but this is not the case when either Safari or Firefox are the host browsers. Both these browsers reset the scroll positions to the top.
+
+As a consequence, once the user scrolls up in the iframe as content updates occur, the programmatic scrolling to the bottom no longer occurs. Instead, the native behavior takes over and keeps the scroll at the top, even as content updates continue.
+
+This makes the {productname} dialog feel stuck when Safari or Firefox are the host browsers, which is both against expected behavior and poor UX.
 
+As of {productname} 6.6.1, the extant `scrollTop` value before any content updates happen is stored. After a content update, if the host browser is Firefox or Safari and the previous scroll position was not at the end of the scrollable area, {productname} now programmatically restore the previous scroll position using the stored `scrollTop` value.
```

Comment:
```suggestion
As of {productname} 6.6.1, after a content update, if the host browser is Firefox or Safari and the previous scroll position was not at the end of the scrollable area, {productname} now programmatically scrolls to the previous position.
```

Reply:
I think it's important to note that we store the existing `scrollTop` value before the content update occurs.
```suggestion
As of {productname} 6.6.1, the extant `scrollTop` value of the iframe dialog component before a content update is stored. After each content update, if the host browser is Firefox or Safari and the previous scroll position was not at the end of the scrollable area, {productname} now programmatically restores the previous scroll position using the stored `scrollTop` value.
```